lab-padget/karma.conf.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
```

Comment:
What's the deal with Karma here in this project?

Just use Mocha and Chai, since we're just testing basics and the overhead and setup are simpler.